### PR TITLE
[Task] request explicit input source

### LINKED ISSUE #114: Replace Request::get with explicit input sources
### Improvement description

Replace Request::get with explicit input sources

### PR DESCRIPTION
**Pull request by mattamon**
Resolves #114

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,9 @@
   "minimum-stability": "dev",
   "require": {
     "pimcore/compatibility-bridge-v10": "^1.0",
-    "pimcore/pimcore": "^10.6 || ^11.0"
+    "pimcore/admin-ui-classic-bundle": "^1.6",
```
Comment (kingjia90) [inline composer.json]:
```suggestion
    "pimcore/admin-ui-classic-bundle": "^1.0",
```
same as in other PRs, there is no hard requirement to a specific minimum version.

Also please remove the entry from the "suggest" section

File: .github/workflows/static-analysis.yml [hunk 2]
```diff
@@ -20,9 +20,9 @@ jobs:
         strategy:
             matrix:
                 include:
-                    - { php-version: "8.0", dependencies: "lowest", pimcore_version: "", phpstan_args: "", experimental: false, require_admin_bundle: false }
                     - { php-version: "8.1", dependencies: "highest", pimcore_version: "", phpstan_args: "", experimental: false, require_admin_bundle: true }
-                    - { php-version: "8.1", dependencies: "highest", pimcore_version: "11.x-dev as 11.0.0", phpstan_args: "", experimental: true, require_admin_bundle: true }
+                    - { php-version: "8.2", dependencies: "highest", pimcore_version: "", phpstan_args: "", experimental: false, require_admin_bundle: true }
+                    - { php-version: "8.3", dependencies: "highest", pimcore_version: "11.x-dev as 11.9.99", phpstan_args: "", experimental: true, require_admin_bundle: true }
```
Comment (kingjia90) [inline .github/workflows/static-analysis.yml]:
```suggestion
                    - { php-version: "8.3", dependencies: "highest", pimcore_version: "11.x-dev", phpstan_args: "", experimental: true, require_admin_bundle: true }
```

**Review by kingjia90 — APPROVED** [review]
Just some indirect changes required after dropping 10, the rest LGTM
Thank you!

File: composer.json (hunk 1 above)
Comment (kingjia90) [reply composer.json]: Also this doc, especially these parts
https://github.com/pimcore/perspective-editor/blob/3114b9dcd206d1c5f0e25cd5169a9aa5a7a9da3e/doc/01_Installation.md?plain=1#L3-L4
https://github.com/pimcore/perspective-editor/blob/3114b9dcd206d1c5f0e25cd5169a9aa5a7a9da3e/doc/01_Installation.md?plain=1#L30-L37

Comment (kingjia90) [reply composer.json]: and if possible also https://github.com/pimcore/perspective-editor/blob/3114b9dcd206d1c5f0e25cd5169a9aa5a7a9da3e/src/Resources/public/js/pimcore/perspective/events.js#L46-L53

File: doc/01_Installation.md [hunk 3]
```diff
@@ -1,49 +1,11 @@
...
-bin/console pimcore:bundle:install PimcorePerspectiveEditorBundle
-```
-
-### For Pimcore 11
-
 You need to follow the steps mentioned above and additionally run the following command:
 
 ```bash
 composer require pimcore/admin-ui-classic-bundle
 ```
```
Comment (kingjia90) [inline doc/01_Installation.md]: this part is not necessary anymore as it is managed by composer

Comment (mattamon) [reply doc/01_Installation.md]: Good catch I also deleted too much

**Review by kingjia90 — APPROVED** [review]

**Pull request merged**